File: fc-chat/demo/part3/ch3/02-2-message-system-add-connection/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/InviteResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.constant.UserConnectionStatus;
import net.prostars.messagesystem.dto.domain.InviteCode;

public class InviteResponse extends BaseMessage {

  private final InviteCode inviteCode;
  private final UserConnectionStatus status;

  public InviteResponse(String type, InviteCode inviteCode, UserConnectionStatus status) {
    super(MessageType.INVITE_RESPONSE);
    this.inviteCode = inviteCode;
    this.status = status;
  }

  public InviteCode getInviteCode() {
    return inviteCode;
  }

  public UserConnectionStatus getStatus() {
    return status;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/InviteResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.constant.MessageType
import com.yscorp.withpush.messagesystem.constant.UserConnectionStatus
import com.yscorp.withpush.messagesystem.dto.domain.InviteCode

class InviteResponse(val inviteCode: InviteCode,val  status: UserConnectionStatus) : BaseMessage(MessageType.INVITE_RESPONSE) {
}
